### DISCUSSION TITLE
highlight vs color

### DISCUSSION THREAD

**Discussion by es183923** [FelixKratz/SketchyBar#60] [Q&A] [answered]
What is the difference between highlight and color?
Doesn't color achieve the same thing as highlight, but a bit more cumbersome, or is that what highlight is for?

**Comment by FelixKratz** [comment, marked as answer]
All texts can have a *color* and a *highlight_color*, because highlighting is a very frequent use case, e.g. for spaces, clicks,.... The rational here is that the color and highlight_color can be set up in the config and do not need to be hardcoded into some script. In the scripts it is possible to turn on and off highlighting via:
```bash
sketchybar -m --set $NAME <icon_highlight/label_highlight>=<on/off/toggle>
```
without specifying the actual colors. This way the configuration for this is kept in the main sketchybarrc. 
E.g. the default space script makes use of this.
Edit: Update to v2.0.0 Syntax
